tests/getAddresses.test.ts

```diff
@@ -55,65 +46,70 @@ describe("listSmartWalletses Tests", () => {
     });
 
     test("should include default smart wallet when authenticated with signature", async () => {
-      const result = await client.listSmartWallets({ authKey });
+      const result = await client.getWallets({ authKey });
       expect(result).toBeDefined();
       expect(result.length).toBeGreaterThanOrEqual(1);
       expect(result[0].salt).toEqual("0");
-      expect(result[0].factory).toEqual("0x29adA1b5217242DEaBB142BC3b1bCfFdd56008e7");
+      expect(result[0].factory).toEqual(FACTORY_ADDRESS);
       expect(result[0].address).toHaveLength(42);
     });
 
     test("should include custom salt wallet when getting address with smartWallet using signature", async () => {
-      await client.createWallet({salt: "12345"}, { authKey });
+      const randomSalt = "12345";
+      await client.createWallet({ salt: randomSalt }, { authKey });
 
-      const wallets = await client.listSmartWallets({ authKey });
+      const wallets = await client.getWallets({ authKey });
       expect(wallets.length).toBeGreaterThanOrEqual(2);
-      expect(wallets.some(item => item.salt === "12345")).toBe(true);
+      expect(wallets.some((item) => item.salt === randomSalt)).toBe(true);
     });
 
     test("create wallet is idempotent", async () => {
-      await client.createWallet({salt: "12345"}, { authKey });
-      await client.createWallet({salt: "12345"}, { authKey });
-      await client.createWallet({salt: 0}, { authKey });
-      await client.createWallet({salt: 0}, { authKey });
-
-      const wallets = await client.listSmartWallets({ authKey });
-      console.log("wallet", wallets);
-      expect(wallets.filter(item => item.salt === "12345")).toHaveLength(1);
-      expect(wallets.filter(item => item.salt === "0")).toHaveLength(1);
+      const salt1 = "12345";
+      const salt2 = "0";
+      const createdWallet1 = await client.createWallet({ salt: salt1 }, { authKey });
+      const createdWallet2 = await client.createWallet({ salt: salt2 }, { authKey });
```

Comment:
@chrisli30 i included multiple calls to prove that this call is idempotent regardless how many time we call it. you remove the call will make us lost that  assertion.

Reply:
oh I see, I recognized those typos ... just added them back.